--- nwchem-6.8/src/atomscf/cguess.fh	2017-12-14 13:36:27.000000000 -0800
+++ patch-files/cguess.fh	2018-04-06 16:14:50.469181273 -0700
@@ -5,7 +5,7 @@
 C$Id$
       integer no                ! Maximum no. of orbitals on any atom ?
       integer maxatsym          ! Maximum no. of syms (s/p/d/f)
-      parameter (nb=500, no=60, maxatsym=4)
+      parameter (nb=600, no=60, maxatsym=4)
 c
       integer nsym              ! highest l-quantum no. used in atomic calc
       integer nbas              ! No. of primitives of each sym
